Sources/MediaViewer/Transitions/MediaViewerInteractivePopTransition.swift

```diff
@@ -198,7 +198,11 @@ extension MediaViewerInteractivePopTransition: UIViewControllerInteractiveTransi
         let navigationBarAlpha = mediaViewer.navigationBarHiddenBackup
         ? 0
         : mediaViewer.navigationBarAlphaBackup
-        animator = UIViewPropertyAnimator(duration: 0.25, dampingRatio: 1) {
+        animator = UIViewPropertyAnimator(
+            duration: 0.25,
+            dampingRatio: 1
+        ) { [weak self] in
```

Comment:
[note]
Normally, the code works correctly even before the fix, but if the user quickly tries to close the screen and methods like `finishInteractiveTransition` are called before `startInteractiveTransition`, the animator’s closure is not released, resulting in a retain cycle.